common/shlibs

```diff
@@ -4165,3 +4166,12 @@ libTMVA.so root-6.24.06_1
 libMathCore.so root-6.24.06_1
 liblowdown.so lowdown-0.11.2_1
 libjaylink.so.0 libjaylink-0.2.0_1
+libomp.so libomp-14.0.3_1
```

Comment:
none of these should be added to shlibs, also libomp needs patching to have a soname (https://raw.githubusercontent.com/chimera-linux/cports/master/main/libomp/patches/0014-add-soname-to-libomp.patch) and a new symlink from libomp.so.* to libomp.so (for devel package) should be installed

Reply:
> a new symlink from libomp.so.* to libomp.so (for devel package) should be installed

what would be the preferred way to do this?

> none of these should be added to shlibs

which ones should I remove, exactly? I'm guessing libomp stays with the patch?